webpack.config.prod.js

```diff
@@ -9,6 +9,14 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = merge(commonConfig, {
     mode: 'production',
+    devtool: '(none)',
```

Comment:
Does it have to say `(none)` or you just omit it?

Reply:
Hm, my .. It just have to be  omitted.  I will remove it.